react-app/src/redux/Login/actions.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-alert */
+import UserService from '../../services/UserService';
+
+const ActionCreators = {
+  submit(values) {
+    return async dispatch => {
+      dispatch({ type: 'GET_USERS' });
```

Comment:
Why are you dispatching this action?

Reply:
This action only exists for informs what is getting data from a server, not executes any other logic. If you consider that I must remove it, I do.